src/pages/api/open-api-docs-preview-v2.ts

```diff
@@ -8,13 +8,12 @@ import path from 'path'
 import { randomUUID } from 'crypto'
 // Types
 import type { NextApiRequest, NextApiResponse } from 'next'
-import { OpenApiPreviewV2InputValues } from 'views/open-api-docs-preview-v2/components/open-api-preview-inputs'
 
 /**
  * Setup for temporary file storage
  */
 // Determine if we're deploying to Vercel, this affects the temporary directory
-const IS_VERCEL_DEPLOY = process.env.VERCEL_ENV !== 'development'
+const IS_VERCEL_DEPLOY = typeof process.env.VERCEL_ENV === 'string'
```

Comment:
Here we fix some `IS_VERCEL_DEPLOY` check logic.

Previously if `VERCEL_ENV` was equal to anything other than `development`, then we'd have `IS_VERCEL_DEPLOY` as `true`. This isn't accurate in cases where `VERCEL_ENV` is undefined, which is sometimes the case when running the site locally.